src/acquisition/flusurv/models.py

```diff
@@ -0,0 +1,29 @@
+from sqlalchemy import Column
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.types import Date, Float, Integer, String
+
+Base = declarative_base()
+
+
+class FluSurv(Base):
```

Comment:
question: does this depend on .sql ddl files or can it replace them?

[src/ddl/fluview.sql](https://github.com/cmu-delphi/delphi-epidata/blob/EPID-42/src/ddl/fluview.sql) is used in two scenarios:

1. (common) setting up the database for the local development environment in Docker
2. (uncommon) setting up a fresh production/staging/QA environment if something has gone Horribly Wrong with one of the existing ones

in both scenarios we want the resulting (empty) database to match what's in (full) prod, with the exception of column order.

with this change, we now have parallel table definitions in fluview.sql and this file. can we drop the .sql definition? if so, what's the procedure for spinning up an empty table based on the sqlalchemy definition?

Reply:
I think it is fully independent

Reply:
ideally we only have one copy of each table definition. if we wanted to drop the .sql file, what would be the procedure for spinning up an empty table based on the sqlalchemy definition?